src/NetAppFiles/NetAppFiles/help/Approve-AzNetAppFilesReplication.md

```diff
@@ -16,19 +16,19 @@ Approve/Authorize replication connection on the source volume
 ```
 Approve-AzNetAppFilesReplication -ResourceGroupName <String> -AccountName <String> -PoolName <String>
  -Name <String> -DataProtectionVolumeId <String> [-PassThru] [-DefaultProfile <IAzureContextContainer>]
- [-WhatIf] [-Confirm] [<CommonParameters>]
+ [-ProgressAction <ActionPreference>] [-WhatIf] [-Confirm] [<CommonParameters>]
```

Comment:
Please only update help files for the cmdlet you changed.

Reply:
Will do.  
But I would argue if the undrelying cmdlet has not changed it should stay the same. 
Also why is the `Update-MarkdownHelpModule` constantly adding this if needs to be removed manually?